extensions/binding/org.eclipse.smarthome.binding.astro/ESH-INF/config/config.xml

```diff
@@ -6,12 +6,9 @@
 
 	<config-description uri="thing-type:astro:config">
 		<parameter name="geolocation" type="text" required="true">
+			<context>location</context>
 			<label>GeoLocation</label>
-			<description>The latitude and longitude separated with a comma (xx.xxxxxx,xx.xxxxxx)</description>
-		</parameter>
-		<parameter name="altitude" type="integer" required="false">
-			<label>Altitude</label>
-			<description>The altitude of the GeoLocation in meter above sea level.</description>
+			<description>The latitude, longitude and altitude separated with a comma (xx.xxxxxx,xx.xxxxxx,xx)</description>
```

Comment:
Assuming that we will soon have a custom widget, I would remove the "(xx.xxxxxx,xx.xxxxxx,xx)" part (or maybe change it to `(<lat>,<long>,<alt>)`.